### DISCUSSION TITLE
Welcome, V205!

### DISCUSSION THREAD

**Discussion by opticaldrive** [ParanoidUser/yolo#1036] [Subscribe]
Can I be a collaborator? Following https://github.com/ParanoidUser/yolo/discussions/26
Thank you.

**Comment by ParanoidUser** [comment]
Ahoy, @V205Arduino!

Thank you for your interest in the project and [welcome aboard](https://github.com/V205Arduino?achievement=pair-extraordinaire&tab=achievements)! 🚢⭐

**Reply by opticaldrive** [reply]
Wait?
How did I become a collaborator?

**Comment by opticaldrive** [comment]
Thank you!
